lib/ansible/plugins/action/script.py

```diff
@@ -76,6 +76,8 @@ def run(self, tmp=None, task_vars=None):
         # out now so we know the file name we need to transfer to the remote,
         # and everything else is an argument to the script which we need later
         # to append to the remote command
+        # FIXME: If the path to the script has a space in it, this silently fails to do anything
+        # This code makes a bad assumption by splitting on spaces.
```

Comment:
we should use 'shell splitting' here ... shlex .... but that is aside from this PR

Reply:
Noted. Thanks for the info.